front/lib/swr/spaces.ts

```diff
@@ -649,11 +651,12 @@ export function useSpaceSearch({
     limit: pagination?.limit ?? DEFAULT_SEARCH_LIMIT,
     query: search,
     viewType,
+    parentId,
   };
 
   // Only perform a query if we have a valid search.
   const url =
-    search.length >= MIN_SEARCH_QUERY_SIZE
+    search.length >= MIN_SEARCH_QUERY_SIZE || parentId
```

Comment:
Note: In my specific case I want to be able to retrieve all children nodes even if the search term length is `< MIN_SEARCH_QUERY_SIZE`.